libs/blocks/table/table.css

```diff
@@ -443,3 +443,19 @@ header.global-navigation {
   width: 30px;
   margin-right: 16px;
 }
+
+/* Start mobile styles */
+@media (max-width: 400px) {
+  .section{
```

Comment:
.section sounds really generic, I'm not sure the table should set something like that

Reply:
Have changed it to a specific selector now.